main.js

```diff
@@ -52,13 +53,17 @@ function createPlayer(player1, name, hp) {
 	const $img = document.createElement('div')
```

Comment:
мы же с тобой обсуждали, не div в img 

```suggestion
	const $img = document.createElement('img')
```